src/confluent_kafka/src/confluent_kafka.c

```diff
@@ -1521,6 +1521,47 @@ static void log_cb (const rd_kafka_t *rk, int level,
         CallState_resume(cs);
 }
 
+static void oauth_cb (rd_kafka_t *rk, const char *oauthbearer_config, void *opaque) {
+        Handle *h = opaque;
+        PyObject *eo, *result;
+        CallState *cs;
+        const char *token;
+        double expiry;
+        char err_msg[2048];
+        rd_kafka_resp_err_t err_code = RD_KAFKA_RESP_ERR_NO_ERROR;
```

Comment:
nit: the initialization is not neededd, it is never used before assignment.

Reply:
My bad, I missed this one from my previous change.